### ISSUE TITLE
game crash

### ISSUE THREAD

**Issue by christomite** [LXGaming/MixinBootstrap#15]
wmy game crashes when i use this mod.

<details>
<summary>debug.log</summary>

```
[27Apr2020 11:45:58.694] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, GOOBERBOMB1, --version, 1.15.2-forge-31.1.51, --gameDir, F:\.minecraft, --assetsDir, F:\.minecraft\assets, --assetIndex, 1.15, --uuid, 34767f4c9680444da461a18c836aff10, --accessToken, ????????, --userType, mojang, --versionType, release, --launchTarget, fmlclient, --fml.forgeVersion, 31.1.51, --fml.mcVersion, 1.15.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20200414.225201]
[27Apr2020 11:45:58.698] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 5.1.0+69+master.79f13f7 starting: java version 1.8.0_51 by Oracle Corporation
[27Apr2020 11:45:59.060] [main/INFO] [net.minecraftforge.fml.loading.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[27Apr2020 11:45:59.741] [main/INFO] [STDOUT/]: [net.minecraftforge.fml.loading.progress.ClientVisualization:initWindow:882]:  BIG FARTS     Corrupt JPEG
[27Apr2020 11:46:00.071] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8 Source=file:/F:/.minecraft/mods/MixinBootstrap-1.0.3.jar Service=ModLauncher Env=CLIENT
[27Apr2020 11:46:00.547] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file Mod File: F:\.minecraft\mods\Chameleon-1.12-4.1.3.jar is missing mods.toml file
[27Apr2020 11:46:00.548] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer/SCAN]: File F:\.minecraft\mods\Chameleon-1.12-4.1.3.jar has been ignored - it is invalid
[27Apr2020 11:46:00.569] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file Mod File: F:\.minecraft\mods\Forgelin-1.8.4.jar is missing mods.toml file
[27Apr2020 11:46:00.569] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer/SCAN]: File F:\.minecraft\mods\Forgelin-1.8.4.jar has been ignored - it is invalid
[27Apr2020 11:46:00.592] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file Mod File: F:\.minecraft\mods\Mixin-master.jar is missing mods.toml file
[27Apr2020 11:46:00.593] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer/SCAN]: File F:\.minecraft\mods\Mixin-master.jar has been ignored - it is invalid
[27Apr2020 11:46:00.593] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file Mod File: F:\.minecraft\mods\MixinBootstrap-master.jar is missing mods.toml file
[27Apr2020 11:46:00.593] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer/SCAN]: File F:\.minecraft\mods\MixinBootstrap-master.jar has been ignored - it is invalid
[27Apr2020 11:46:01.228] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 11:99 token recognition error at: ';'
[27Apr2020 11:46:01.278] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:0 token recognition error at: 'S'
[27Apr2020 11:46:01.278] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:1 token recognition error at: 'pa'
[27Apr2020 11:46:01.278] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:3 token recognition error at: 'w'
[27Apr2020 11:46:01.279] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:4 token recognition error at: 'n'
[27Apr2020 11:46:01.279] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:5 token recognition error at: 'e'
[27Apr2020 11:46:01.279] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:6 token recognition error at: 'r'
[27Apr2020 11:46:01.279] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:8 token recognition error at: 'N'
[27Apr2020 11:46:01.279] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:9 token recognition error at: 'B'
[27Apr2020 11:46:01.280] [main/INFO] [STDERR/]: [org.antlr.v4.runtime.ConsoleErrorListener:syntaxError:38]: line 36:10 token recognition error at: 'T'
[27Apr2020 11:46:02.308] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [snownee.fruits.mixin.Connector]
[27Apr2020 11:46:02.310] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [me.jellysquid.mods.phosphor.mixin.PhosphorMixinConnector]
[27Apr2020 11:46:02.312] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [snownee.snow.mixin.Connector]
[27Apr2020 11:46:02.316] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [com.stevekung.stevekungslib.core.SteveKunGLibMixinConnector]
[27Apr2020 11:46:02.319] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [com._13rac1.erosion.forge.MixinConnector]
[27Apr2020 11:46:02.324] [main/INFO] [Fruit Trees/]: Invoking Mixin Connector
[27Apr2020 11:46:02.380] [main/INFO] [Kiwi/]: Invoking Mixin Connector
[27Apr2020 11:46:02.388] [main/WARN] [mixin/]: Mixin config mixins.stevekung's_lib.json requires mixin subsystem version 0.8.1 but 0.8 was found. The mixin config will not be applied.
[27Apr2020 11:46:02.389] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: org.spongepowered.asm.launch.MixinInitialisationError: Required mixin config mixins.stevekung's_lib.json requires mixin subsystem version 0.8.1
[27Apr2020 11:46:02.389] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.transformer.MixinConfig.checkVersion(MixinConfig.java:574)
[27Apr2020 11:46:02.389] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.transformer.MixinConfig.postInit(MixinConfig.java:463)
[27Apr2020 11:46:02.389] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.transformer.MixinConfig.onLoad(MixinConfig.java:390)
[27Apr2020 11:46:02.389] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.transformer.MixinConfig.create(MixinConfig.java:1085)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.transformer.Config.create(Config.java:148)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.Mixins.createConfiguration(Mixins.java:100)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.mixin.Mixins.addConfiguration(Mixins.java:87)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at com.stevekung.stevekungslib.core.SteveKunGLibMixinConnector.connect(SteveKunGLibMixinConnector.java:13)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.launch.platform.MixinConnectorManager.initConnectors(MixinConnectorManager.java:96)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.launch.platform.MixinConnectorManager.inject(MixinConnectorManager.java:61)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.launch.platform.MixinPlatformManager.inject(MixinPlatformManager.java:197)
[27Apr2020 11:46:02.390] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.launch.MixinBootstrap.inject(MixinBootstrap.java:190)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at org.spongepowered.asm.launch.MixinLaunchPlugin.initializeLaunch(MixinLaunchPlugin.java:194)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.LaunchPluginHandler.lambda$announceLaunch$12(LaunchPluginHandler.java:95)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.LaunchPluginHandler$$Lambda$430/1845922505.accept(Unknown Source)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.util.HashMap.forEach(HashMap.java:1280)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.LaunchPluginHandler.announceLaunch(LaunchPluginHandler.java:95)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:52)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81)
[27Apr2020 11:46:02.391] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)
```

**Comment by LXGaming** [comment]
Update [SteveKunG's Lib](https://www.curseforge.com/minecraft/mc-mods/stevekungs-lib), This issue was fixed in [2.0.2](https://github.com/SteveKunG/SteveKunG-Lib/commit/1851c6cc8dada49037e3700e375bb4e200d36b9e#diff-4b6e7df858ebf0bb1573847df77e12ff)

**Comment by christomite** [comment]
it worked thank you so very much!!!
